web/src/navbar_alerts.ts

```diff
@@ -375,6 +385,10 @@ export function initialize(): void {
         );
     });
 
+    $("#navbar_alerts_wrapper").on("click", ".download_latest_zulip_version", () => {
```

Comment:
You should use dashes for all the new class names like `download_latest_zulip_version` that you're introducing.

Reply:
Oh, I was under the impression that we should be using snake_case for classes that are only being used for JS logic and don't have any CSS styling attached to them.

Reply:
Updated the class names to use dashes instead of the snake case.

Reply:
Nope, it's just a general naming convention preference, but we've not made any attempt to bulk-migrate them all, because that's likely to cause a lot of breakage.